dynlink.raml

```diff
@@ -0,0 +1,216 @@
+                        "ins": "f33a89b2"
+                      },
+                      {
+                        "href": "scenevalue",
+                        "rel": ["boundto"],
+                        "bind": "push",
+                        "anchor": "ocf://5c68f373-0c4c-4604-9922-db639e20533d/heatcontrol",
+                        "if": ["oic.if.a"],
+                        "ins": "ca5f1004"
+                      },
+                      {
+                        "href": "377fac40",
+                        "rt": "oic.r.linkstatus",
+                        "if": ["oic.if.baseline", "oic.if.rw"],
+                        "linkinstance": "377fac40"
+                      },
+                      {
+                        "href": "f33a89b2",
+                        "rt": "oic.r.linkstatus",
+                        "if": ["oic.if.baseline", "oic.if.rw"],
+                        "linkinstance": "f33a89b2"
+                      },
+                      {
+                        "href": "ca5f1004",
+                        "rt": "oic.r.linkstatus",
+                        "if": ["oic.if.baseline", "oic.if.rw"],
+                        "linkinstance": "ca5f1004"
+                      }
+                    ]
+
+
+/DynlinkResURI?if=oic.if.b:
+    description: |
+      Batch representation of a dynlink contains the link status resources
+    displayName: dynlink-batch
+
+    get:
+        description: |
+          Batch interface of the dynlink resource type
+        responses:
+          200:
+            body:
+              application/json:
+                schema: batch
+                  example: |
+                    [
+                      {
+                        "href": "377fac40",
+                        "rep": {
+                          "linkstate": "active",
+                          "laststatus": "200",
+                          "endtime": "2018-06-20T07:07:03+00:00",
+                          "maxwait": 6000,
+                          "linkinstance": "7ffe23ac"
+                        }
+                      },
+                      {
+                        "href": "f33a89b2",
+                        "rep": {
+                          "linkstate": "active",
+                          "laststatus": "200",
+                          "endtime": "2018-06-20T07:07:03+00:00",
+                          "maxwait": 6000,
+                          "linkinstance": "f33a89b2"
+                        }
+                      },
+                      {
+                        "href": "ca5f1004",
+                        "rep": {
+                          "linkstate": "active",
+                          "laststatus": "200",
+                          "endtime": "2018-06-20T07:07:03+00:00",
+                          "maxwait": 6000,
+                          "linkinstance": "ca5f1004"
+                        }
+                      }
+                    ]
+
```

Comment:
Why doesn't the "batch" response include a representation of the Dynamic Link Resources?  They are in the Collection so they should be included, right?